Update `Local song offset` (#8787)
ppy/osu-wiki: wiki/Glossary/Local_song_offset/en.md

*For offset while beatmapping, see: [Offset](/wiki/Beatmapping/Offset)*

> **Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmap). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the global offset to calculate a total [offset](/wiki/Beatmapping/Offset).

Reviewer: the link is incorrect, that page is not about a "total" offset.

```suggestion
**Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmap). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the [global offset](/wiki/Client/Options/Universal_offset) to calculate a total offset.
```

- `Alt` + `+` to increase the offset by 1 milliseconds
- `Alt` + `-` to decrease the offset by 1 milliseconds

If a local offset is defined, osu! will display the local offset in the interface above the scoreboard.

> ![Local Offset](/wiki/Glossary/img/Local_Offset.jpg "Local Offset")

Reviewer: ```suggestion
![Local offset](img/local-offset.jpg "Local offset")
```

osu! will also notify you of the local offset before starting gameplay.

> ![Local Offset Notice](/wiki/Glossary/img/Local_Offset_Notice.jpg "Notice of Offset change")

Reviewer: ```suggestion
![Local offset notice](img/local-offset-notice.jpg "Notice of offset change")
```

> <!-- TODO: Add links and stuff -->

Reviewer 2: I assume you have completed the TODO with your changes
